sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/test/scala/com/azure/cosmos/spark/CosmosClientCacheSpec.scala

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.spark
+
+import com.azure.cosmos.CosmosAsyncClient
+import com.azure.cosmos.implementation.{CosmosClientMetadataCachesSnapshot, TestConfigurations}
+import org.apache.spark.broadcast.Broadcast
+import org.mockito.Mockito.{mock, verify}
+
+class CosmosClientCacheSpec extends UnitSpec {
+    //scalastyle:off multiple.string.literals
+
+    private val cosmosEndpoint = TestConfigurations.HOST
```

Comment:
the indentation is not correct. Please check other tests. we should use 2 spaces for indentation to be consistent with other tests.

Reply:
Right, but the indentation configuration in the repo is for 4 spaces, so IntelliJ uses that

Reply:
> Right, but the indentation configuration in the repo is for 4 spaces, so IntelliJ uses that

weird, for scala code my intellj shows 2 space indentation as default everywhere.
where are you getting the 4 space configuration from?

Reply:
Found the problem, something was overriding, I reseted to default